pom.xml

```diff
@@ -94,6 +93,12 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.21</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
It looks like the project compiles just fine without this. Is there a specific reason to declare a test-scope dependency on slf4j?